### DISCUSSION TITLE
New meeting request page

### DISCUSSION THREAD

**Discussion by marvabban** [code4recovery/12-step-meeting-list#953] [Q&A] [answered]
Is there a "Request a new meeting" page? I used the "meeting list feedback" plugin and it works great for requesting changes to existing meetings. It also has a "New meeting" option, but you have to navigate to an existing meeting's details page. Is there a way to have a "Request a new meeting" form on its own page?

**Comment by kiyote33** [comment, marked as answer]
Hi @marvabban Mark,
Unfortunately, at this time there is not a separate New Meeting page, we simply transform the same page depending on whether or not the user is making a "New" or "Change" request. What others with the same issue have done is to employ one of the widgets (like "Meeting Detail Bottom") to post a how-to information message on navigating to and using the radio buttons to make your request type choice. 
Hope this helps...

**Reply by marvabban** [reply]
Thanks for the feedback and all the excellent work you and your team do with this plugin.
